satpy/readers/_geos_area.py

```diff
@@ -166,3 +165,61 @@ def sampling_to_lfac_cfac(sampling):
         Line/column scaling factor (deg-1)
     """
     return 2.0 ** 16 / np.rad2deg(sampling)
+
+
+def get_geos_area_naming(input_dict):
+    """Get a dictionary containing formatted AreaDefinition naming.
+
+    Args:
+        input_dict: dict
+            Dictionary with keys `platform_name`, `instrument_name`, `service_name`, `resolution` .
```

Comment:
`service_desc` is missing as key. But apart form that very minor point, it looks very good 👍

Reply:
Thanks a lot for spotting this! Fixed now.